cmd/podman/create.go

```diff
@@ -88,12 +106,17 @@ func createCmd(c *cli.Context) error {
 	rtc := runtime.GetConfig()
 	ctx := getContext()
 
-	newImage, err := runtime.ImageRuntime().New(ctx, c.Args()[0], rtc.SignaturePolicyPath, "", os.Stderr, nil, image.SigningOptions{}, false, false)
-	if err != nil {
-		return err
+	imageName := ""
```

Comment:
(From a quick search this will probably mess up `podman ps` formatting.)

Reply:
Agree. My preference is probably to manually handle the no-image case and pop up a `<none>` or similar to indicate it's not using an image